### DISCUSSION TITLE
Migrate agents from Eval to Standalone?

### DISCUSSION THREAD

**Discussion by Odiesinfosec** [Security-Onion-Solutions/securityonion#10342] [Unsupported Versions]
Hello, 
we have been using Evaluation for about 2 months now, and we have decided we love it & to dedicate more resources with the standalone version! 
(we have >100 hosts + servers and barely scratch 10MB throughput I think standalone will be fine) 
My question is, Will I have to completely scrap everything and start from new, or is there a way to migrate?
I am asking this because I have already installed Wazuh and Osquery through GPO. I have tried to uninstall software with GPO in the past and it rarely ever works out, would be really awesome to have a way to migrate or inform the agents of the new host, as well as keep my data from the Evaluation. 
Is it possible or will I have a long week ahead of me by manually uninstalling the old agents? 
even if I can just adjust the Osquery already installed to work with the new standalone so2...
Cheers

**Comment by dougburks** [comment]
Security Onion 2.4 (currently in Beta) replaces Wazuh and Osquery with the Elastic Agent which is easier to manage. You may just want to consider keeping your deployment as-is until 2.4 releases and then switch everything over.
https://blog.securityonion.net/2023/04/security-onion-24-beta-2-release-now.html
